extension/content.js

```diff
@@ -268,7 +268,11 @@ function linkifyIssuesInTitles() {
 }
 
 function addPatchDiffLinks() {
-	const commitUrl = location.href.replace(/\/$/, '');
+	let commitUrl = location.href.replace(/\/$/, '');
+	const prCommit = /\/pull\/\d+\/commits/;
+	if (prCommit.test(commitUrl)) {
+		commitUrl = commitUrl.replace(prCommit, '/commit');
```

Comment:
This check does not handle forks well. I'm not sure if it's easy to support forks in this scenario though.